control/controlclient/map.go

```diff
@@ -778,14 +753,19 @@ func peerChangeDiff(was tailcfg.NodeView, n *tailcfg.Node) (_ *tailcfg.PeerChang
 	return ret, true
 }
 
+func (ms *mapSession) sortedPeers() []tailcfg.NodeView {
```

Comment:
If we wanted to keep the memorized version of this, we could put it in a lazy value and clear it any time the peer list is written to